src/ledger/utils.ts

```diff
@@ -74,7 +74,7 @@ function renameCounterpartyToIssuerInOrder(order: RequestBookOffersArgs) {
   const taker_gets = renameCounterpartyToIssuer(order.taker_gets)
   const taker_pays = renameCounterpartyToIssuer(order.taker_pays)
   const changes = {taker_gets, taker_pays}
-  return _.assign({}, order, _.omitBy(changes, _.isUndefined))
+  return Object.assign({}, order, _.omitBy(changes, value => value === undefined))
```

Comment:
Looking at `_.forEach(...)`, replaced with `Object.entries(...).forEach([_, ...])`.

Is it possible to to `EmitBy` with something like:

```js
const omitted = Object.entries(...).filter([_, ...] => value !== undefined)`

Object.assign({}, order, Object.fromEntries(omitted))
```

It seems strange to be removing one lodash function from this line, but leaving the other.

Reply:
Yeah that's fair, I didn't want to get rid of all the methods and unless I remove all the `omitBy` calls we don't get any saved space.

Reply:
Oh, yeah, that makes sense. Unless you wanna rip it out entirely, then I'm okay w/ leaving it.

Reply:
I want to see what lodash methods we still use in the new version, I don't think it's worth doing any additional refactoring rn